src/rsdp_search.rs

```diff
@@ -0,0 +1,44 @@
+use core::mem;
+use AcpiHandler;
+
+/// The pointer to the EBDA (Extended Bios Data Area) start segment pointer
+pub const EBDA_START_SEGMENT_PTR: usize = 0x40e;
```

Comment:
If we move all the relevant functions into this module, there's no reason to keep these public